aidants_connect_web/static/css/mandat.css

```diff
@@ -1,51 +1,74 @@
-.hero__container {
-  min-height: 10em;
+form {
+    max-width: 40em;
 }
 
-ul {
-  line-height: 1.875em;
-  margin: 0;
+.tile p {
+    font-weight: bold;
 }
 
-form ul {
-  list-style-type: none;
+.tile label {padding: 12px 30px;
+    width: 100%;
+    display: block;
+    text-align: left;
+    color: black;
+    cursor: pointer;
+    position: relative;
+    z-index: 2;
+    transition: color 100ms ease-in;
+    overflow: hidden;
 }
 
-ul input[type="radio"] {
-  margin-bottom: 0.25em;
+.tile label::before {
```

Comment:
I'm not exactly sure why use a `::before` here, it seems that it brings complexity and might cause the spacing issues

Reply:
I would not know how to replace it, I hope you can show me.